### PR TITLE
Create an RPM for an Alpine Linux LXC container.

### PR DESCRIPTION
**Pull request by mliudev**

### CODE REVIEW

**Comment by CtrlC-Root** [comment]
The lxc-sysop-alpine template should also configure the machine hostname using the `utsname` configuration setting.

File: packages/lxc-sysop-alpine-3.3/sysop-alpine.sh [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+# Common container settings
+
+EOF
+
+cat $SYSOP_CONFIG >> $CONFIG_TEMP
+cat <<EOF >> $CONFIG_TEMP
+
+# Container specific settings
+
```
Comment (CtrlC-Root) [inline packages/lxc-sysop-alpine-3.3/sysop-alpine.sh]: get rid of this line

**Comment by mliudev** [comment]
@CtrlC-Root Implemented review suggestions.

**Comment by CtrlC-Root** [comment]
:shipit:

**Pull request merged**